src/Service/ExcelImport.php

```diff
@@ -96,15 +96,19 @@ public function importExcel(string $excelFilePath): LsDoc
 
         $sheet = $phpExcelObject->getSheetByName('CF Association');
         $lastRow = $sheet->getHighestRow();
+        $associationsIdentifiers = [];
+
         for ($i = 2; $i <= $lastRow; ++$i) {
             $association = $this->saveAssociation($sheet, $doc, $i, $items, $children);
+            $associationsIdentifiers[$this->getCellValueOrNull($sheet, 1, $i)] = null;
+
             if (null !== $association) {
```

Comment:
This block, and the `$associations` array, does not appear to be used anymore and should probably be removed.

Reply:
you're right, I just removed it